.github/workflows/ci.yml

```diff
@@ -122,20 +105,16 @@ jobs:
           submodules: "recursive"
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v2
-      - name: Install Pixi
+      - uses: prefix-dev/setup-pixi@v0.8.1
+        with:
+          activate-environment: true
+          cache: true
+          cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
+          manifest-path: build/pixi.toml
+      - name: Tweak environment to find GDAL
         run: |
-          curl -fsSL https://pixi.sh/install.sh | bash
-          echo "$HOME/.pixi/bin" >> $GITHUB_PATH
-          echo "GDAL_HOME=$(pwd)/build/.pixi/envs/default" >> "$GITHUB_ENV"
-          echo "LD_LIBRARY_PATH=$(pwd)/build/.pixi/envs/default/lib" >> "$GITHUB_ENV"
-          echo "GEOS_LIB_DIR=$(pwd)/build/.pixi/envs/default/lib" >> "$GITHUB_ENV"
```

Comment:
I do think you probably need `GDAL_HOME` and `GEOS_LIB_DIR` manually set

Reply:
Seems to be passing w/o them? Right now only failure is on the Python .so